.env

```diff
@@ -1,2 +1,2 @@
-ACCESS_TOKEN = "yIR9I7shhcWPnH6AUlvmb7tryAfgNDEHI3GjxUA6uWs"
```

Comment:
this is BAD, how could `.env` files be found in a commit? It was supposed to be ignored.

Reply:
alright boss...